src/routes/safe/components/Transactions/TxList/TxSingularDetails.tsx

```diff
@@ -49,15 +36,7 @@ const TxSingularDetails = (): ReactElement => {
     shallowEqual,
   )
 
-  // The indexedTx can be temporailiy not found when re-fetching the queue
-  // To avoid showing a loader, we use a locally cached version of it
-  useEffect(() => {
-    if (indexedTx != null) {
-      setLiveTx(indexedTx)
```

Comment:
We no longer need to cache the `liveTx` as we now cache the queued transactions, on top of fetching the queue when a transaction is created.